### PR TITLE
Feat/blockchain explorer

### PR DESCRIPTION
**Pull request by ClemensLey**

### CODE REVIEW

File: packages/explorer/scripts/deploy.mjs [hunk 1]
```diff
@@ -38,49 +38,72 @@ Do you want to deploy the contracts? (y/n)
...
+        }
+      }
+
+      console.log("Creating course on operational semantics with instructor Peter Landin")
+      const course = await computer.new(Course, ["Operational Semantics", peter])
+
+      console.log("Adding student Alan Turing")
+      await course.addStudent(alan)
+
+      // console.log("Adding student Satoshi Nakamoto")
```
Comment (ltardivo) [inline packages/explorer/scripts/deploy.mjs:75]: Should we remove the commented code?

File: packages/explorer/src/components/ModalOld.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import { Link, useNavigate } from "react-router-dom"
```
Comment (ltardivo) [inline packages/explorer/src/components/ModalOld.tsx:1]: Is this file still needed?

File: packages/explorer/src/components/ModalTemp.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
+import { Dispatch, SetStateAction, useEffect, useState } from "react"
```
Comment (ltardivo) [inline packages/explorer/src/components/ModalTemp.tsx:1]: Is this file still needed?

File: packages/explorer/src/components/Wallet.tsx [hunk 4]
```diff
@@ -14,74 +14,101 @@ export default function Wallet2({ computer }: { computer: Computer }) {
...
+  return <div className="mb-6">
+      <span className="block mb-1 text-sm font-medium text-gray-900 dark:text-white">
+        Balance
+        <HiRefresh
+          onClick={refreshBalance}
+          className="w-4 h-4 ml-1 inline cursor-pointer text-gray-500 dark:text-gray-400 hover:text-slate-700 dark:hover:text-slate-100"
+        />
+      </span>
+      <p className="mb-4 font-mono text-xs text-gray-500 dark:text-gray-400">
+        {balance / 1e8} LTC
```
Comment (ltardivo) [inline packages/explorer/src/components/Wallet.tsx:26]: We should read the coin from the login, now that we have that parametrized.

File: packages/explorer/src/components/playground/Playground.tsx [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+            Deploy Modules
+          </button>
+        </li>
+      </ul>
+    </div>
+  )
+}
+
+const Playground = () => {
+  console.log('Playground')
```
Comment (ltardivo) [inline packages/explorer/src/components/playground/Playground.tsx:128]: remove console.log

File: packages/explorer/src/components/playground/Playground.tsx [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+            />
+          </div>
+        </div>
+
+        <hr className="h-px my-8 bg-gray-200 border-0 dark:bg-gray-700" />
+
+        <Examples loadExamples={loadExamples} clearExamples={clearExamples} />
+
+      </div>
+      <ModalOld
```
Comment (ltardivo) [inline packages/explorer/src/components/playground/Playground.tsx:231]: Here we are using the ModalOld, maybe we should change the name, or use the new Modal component

File: packages/explorer/src/utils.ts [hunk 7]
```diff
@@ -2,3 +2,116 @@ export const getFnParamNames = function (fn: string) {
...
+  }
+
+export const strip = (value: Json): Json => {
+  if (isJBasic(value)) return value
+  if (isJArray(value)) return value.map(strip)
+  const { _id, _root, _rev, _amount, _owners, ...rest } = value
+  return rest
+}
+
+export const chunk = (arr: string[], chunkSize = 4): string[][] => {
```
Comment (ltardivo) [inline packages/explorer/src/utils.ts:66]: We should remove the functions that are not used, eg. chunk, strip, etc

**Comment by ClemensLey** [comment]
Great!
